### ISSUE TITLE
Error Getting platform_full_version for Apple M1 Computer

### ISSUE THREAD

**Issue by kmlefran** [cclib/cclib#1420]
Trying to parse Gaussian log files generated on an Apple M1 Computer generates an error. 

```
_, _, platform_full_version, compile_date = next(inputfile).split()
ValueError: too many values to unpack (expected 4)
```

This is because the computer name is two words in the Gaussian log file. On the line after the citation, shown below, splitting the line results in 5 words instead of the expected 4
```
Gaussian 16:  Apple M1-G16RevC.02  7-Dec-2021
```

Error is not observed if the line is manually corrected to: (note removal of space between Apple and M1)

```
Gaussian 16:  AppleM1-G16RevC.02  7-Dec-2021
```

Is it possible to get a fix for this, rather than having to adjust all files manually?

**Comment by berquist** [comment]
Thanks for letting us know.  Do you have an output file that can be shared so we can test against a proper fix?

**Comment by kmlefran** [comment]
Here is a Gaussian file affected by this issue

[co_pbe1pbe_631ppGss.log](https://github.com/cclib/cclib/files/15151069/co_pbe1pbe_631ppGss.log)

**Comment by berquist** [comment]
Ok, I have a proper fix.  Once this issue is closed, it will be in the `master` branch.  If you want to install it, `python -m pip install git+https://github.com/cclib/cclib.git@master` will work, with a caveat: there have been some breaking changes made to `master` since 1.8.1 was released, the most notable is units (all energies are now in hartree).  The fix will be in 2.0, which will have more breaking (API) changes, but will take some months to be out.
